### DISCUSSION TITLE
How to create a release

### DISCUSSION THREAD

**Discussion by james-s-w-clark** [BrianLima/UWPHook#122] [Ideas]
Hey, I built a debug/DefaultBuild/Release app locally - but I'm wondering how forks could make releases w/ the Installer setup etc. (so any users of the forked code don't need to install Visual Studio etc.).

Also, I am wondering if GitHub Actions making a release for you (e.g. on tag creation) is a nice thing to have.

Great app :)

**Comment by BrianLima** [comment]
Hey, releases are built with NSIS, it basically zips the built release files and creates some directories, so installing the NSIS application and running the script, it depends on some hardcoded paths, but it should be easy to be adapted.
